app/controllers/amr_heatmap_controller.rb

```diff
@@ -0,0 +1,39 @@
+class AmrHeatmapController < ApplicationController
+  include ReportHelper
+
+  before_action :admin_required
+
+  power :viewable_samples
+
+  # GET /amr_heatmap.json
+  # Return JSON information required to create a visualization,
+  # from submitted parameters
```

Comment:
I think it's better if this comment doesn't mention the visualization (since the amr_counts could conceivably be used for other purposes), and instead elaborates a little on what `amr_counts` is and the fact that this endpoint receives multiple samples.

Reply:
Done! Updated with better comment.